pkgs/data/fonts/adwaita-fonts/default.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  pkg-config,
+  meson,
+  ninja,
+  glib,
+  gnome-common,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "adwaita-fonts";
+  version = "48.2";
+
+  src = fetchzip {
+    url = "https://gitlab.gnome.org/GNOME/adwaita-fonts/-/archive/${version}/adwaita-fonts-${version}.tar.gz";
+    sha256 = "sha256-rXr4U5k0MUz766F5kVssZfM6Ra/hQOe/HLpGss2aZuo=";
+  };
```

There is a fetcher for GitLab, you may want to use it instead of `fetchzip`:
```suggestion
  src = fetchFromGitLab {
    domain = "gitlab.gnome.org";
    owner = "GNOME";
    repo = "adwaita-fonts";
    rev = "${version}";
    hash = "";
  };
```

---

ok